wire-kotlin-generator/src/main/java/com/squareup/wire/kotlin/UnknownFieldBuilder.kt

```diff
@@ -0,0 +1,82 @@
+/*
+ * Copyright 2018 Square Inc.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ * http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+package com.squareup.wire.kotlin
+
+import com.squareup.wire.FieldEncoding
+import com.squareup.wire.ProtoAdapter
+import com.squareup.wire.ProtoReader
+import com.squareup.wire.ProtoWriter
+import okio.Buffer
+import okio.ByteString
+
+class UnknownFieldsBuilder {
```

Comment:
I wonder if we should migrate this to Java so the runtime doesn’t have a Kotlin dependency

Reply:
`KotlinGenerator.kt` is in kotlin, will migrating only this file help ?

Reply:
Only the unknown fields builder needs to be migrated. It's the only file in the runtime package.